module/VuFind/src/VuFind/View/Helper/AbstractSearch.php

```diff
@@ -77,10 +77,10 @@ public function renderSpellingSuggestions($msg, $results, $view)
         }
 
         $html = '<div class="' . $this->getContainerClass() . '">';
-        $html .= '<h2 class="spellingSuggestions">' . $msg . '</h2>';
+        $html .= '<h2 class="spellingSuggestions">' . $msg . '</h2><ul>';
```

Comment:
On second thought, maybe this should be an ordered `<ol>` since it matches the order of the search terms?

Reply:
The other question is whether the inner list of suggestions should also be a list! Because we have an outer list of input terms, and then each input term has its own independent list of suggestions. I think you could argue for ordered or unordered lists in each case as well. There is an order to both (input order for the input list, and relevance order for the output list) but it's not especially obvious and may not mean much to the end user.

Reply:
I agree -- the inner list *should* be a list as well.

I think I'm going to skip the ordered lists, although it's correct in some sense, it won't be obvious to the user.